frontend/public/components/utils/headings.tsx

```diff
@@ -196,23 +199,32 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
               <span className="co-m-pane__heading-badge">{badge}</span>
             )}
             {link && <div className="co-m-pane__heading-link">{link}</div>}
-            {showActions && (
+            {(perspective === 'admin' || showActions) && (
```

Comment:
Do we need `perspective === 'admin'` checks as we have the same checks below at line no. 204?

Reply:
Yes it is required because we don't want to create unnecessary div (line no 203) if perspective is not admin and `showActions` is not defined.